entity-service-impl/src/main/java/org/hypertrace/entity/metric/EntityMetricsReporter.java

```diff
@@ -1,18 +1,81 @@
 package org.hypertrace.entity.metric;
 
+import static org.hypertrace.core.documentstore.expression.operators.AggregationOperator.COUNT;
+import static org.hypertrace.core.documentstore.model.config.CustomMetricConfig.VALUE_KEY;
+
+import java.time.Duration;
+import java.util.List;
 import org.hypertrace.core.documentstore.Datastore;
+import org.hypertrace.core.documentstore.expression.impl.AggregateExpression;
+import org.hypertrace.core.documentstore.expression.impl.ConstantExpression;
+import org.hypertrace.core.documentstore.expression.impl.IdentifierExpression;
+import org.hypertrace.core.documentstore.expression.impl.LogicalExpression;
+import org.hypertrace.core.documentstore.expression.impl.RelationalExpression;
+import org.hypertrace.core.documentstore.expression.operators.RelationalOperator;
+import org.hypertrace.core.documentstore.model.config.CustomMetricConfig;
+import org.hypertrace.core.documentstore.query.Filter;
+import org.hypertrace.core.documentstore.query.Query;
+import org.hypertrace.core.serviceframework.docstore.metrics.DocStoreCustomMetricReportingConfig;
 import org.hypertrace.core.serviceframework.docstore.metrics.DocStoreMetricsRegistry;
 import org.hypertrace.core.serviceframework.spi.PlatformServiceLifecycle;
+import org.hypertrace.entity.v1.entitytype.EntityType;
 
 public class EntityMetricsReporter {
+
+  private static final String ALL_API_COUNT_METRIC_NAME = "all.api.entities.count";
```

Comment:
Given that the metric changed now, can we now name the metric differently?
```suggestion
  private static final String API_COUNT_METRIC_NAME = "api.entities.count";
```